pkg/controller/servicebindingrequest/envvars/envvars_test.go

```diff
@@ -0,0 +1,47 @@
+package envvars
+
+import (
+	"bytes"
+	"testing"
+	"text/template"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuild(t *testing.T) {
+	src := map[string]interface{}{
+		"status": map[string]interface{}{
+			"listeners": []map[string]interface{}{
+				{
+					"type": "secure",
+					"addresses": []map[string]interface{}{
+						{
+							"host": "my-cluster-kafka-bootstrap.coffeeshop.svc",
+							"port": "9093",
+						},
+					},
+				},
+			},
+		},
+	}
+
+	expected := map[string]string{
+		"STATUS_LISTENERS_0_TYPE":             "secure",
+		"STATUS_LISTENERS_0_ADDRESSES_0_HOST": "my-cluster-kafka-bootstrap.coffeeshop.svc",
+		"STATUS_LISTENERS_0_ADDRESSES_0_PORT": "9093",
+	}
+
+	actual, err := Build(src, []string{})
```

Comment:
add a test case when path is not empty.

Reply:
Increase coverage in existing tests #471